package.json

```diff
@@ -32,11 +32,11 @@
     "clean": "rm -rf dist/*",
     "prepare": "npm run build",
     "check:client:tsc": "npx tsc --p client/tsconfig.json --noEmit",
-    "build:server": "npx tsc --build server/tsconfig.json",
-    "build:client": "node ./scripts/build.js",
+    "build:server": "npx tsc --build tsconfig.json",
+    "build:client": "node ./scripts/build.mjs",
```

Comment:
Why have you changed the file type to a mjs?

This is a script, not a module. Is there a good reason that i missed?